lib/dataset/query.js

```diff
@@ -338,6 +338,50 @@ define({
             return this.mergeOptions({distinct: args});
         },
 
+        /*   {
+         *     division: db.from('division').one();
```

Comment:
Hmm this seems weird to pass in an action directly, I suppose you could but we should also accept a function to do delayed init that way results are refreshed everytime its called. My concern would be having a base Dataset that we use in multiple places if we accepted functions we could reuse it ensuring we get the latest each time. 

I think most of the time this will be used in tandem with associations and I think it could be important there also.